SerialPrograms/Source/PokemonSV/Options/PokemonSV_BBQOption.h

```diff
@@ -0,0 +1,80 @@
+/*  BBQ Option
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#ifndef PokemonAutomation_PokemonSV_BBQOption_H
+#define PokemonAutomation_PokemonSV_BBQOption_H
+
+#include "Common/Cpp/Options/StaticTextOption.h"
+#include "Common/Cpp/Options/GroupOption.h"
+#include "Common/Cpp/Options/EnumDropdownOption.h"
+#include "Common/Cpp/Options/SimpleIntegerOption.h"
+#include "Common/Cpp/Options/BooleanCheckBoxOption.h"
+#include "CommonFramework/Options/LanguageOCROption.h"
+#include "PokemonSwSh/Options/PokemonSwSh_BallSelectOption.h"
+#include "PokemonSV/Options/PokemonSV_BattleMoveTable.h"
+#include "PokemonSV/Options/PokemonSV_TeraAIOption.h"
+#include "PokemonSV/Options/PokemonSV_TeraCatchOnWinOption.h"
+
+namespace PokemonAutomation{
+namespace NintendoSwitch{
+namespace PokemonSV{
+
+class BBQOption : public GroupOption, private ConfigOption::Listener {
```

Comment:
Do you need to inherit from `ConfigOption::Listener`? I don't see an override for any of its functions.